<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>502</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_inputMatrix">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;Input Matrix&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTableView" name="tableView_inputMatrix"/>
    </item>
    <item>
     <widget class="QLabel" name="label_outputMatrix">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;Output Matrix&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTableView" name="tableView_outputMatrix"/>
    </item>
    <item>
     <widget class="QLabel" name="label_methodResults">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;Method Results&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="plainTextEdit_methodResults">
      <property name="lineWrapMode">
       <enum>QPlainTextEdit::NoWrap</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar_Main">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>570</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTests">
    <property name="title">
     <string>tests</string>
    </property>
    <addaction name="action_init"/>
    <addaction name="action_test"/>
    <addaction name="action_numPy"/>
    <addaction name="action_algorithmData"/>
    <addaction name="action_algorithmInput"/>
    <addaction name="action_charInput"/>
    <addaction name="action_plot"/>
   </widget>
   <addaction name="menuTests"/>
  </widget>
  <widget class="QStatusBar" name="statusbar_Main"/>
  <widget class="QToolBar" name="toolBar_Main">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_init"/>
   <addaction name="action_test"/>
   <addaction name="action_numPy"/>
   <addaction name="action_algorithmData"/>
   <addaction name="action_charInput"/>
   <addaction name="action_algorithmInput"/>
   <addaction name="action_plot"/>
  </widget>
  <action name="action_test">
   <property name="text">
    <string>Test</string>
   </property>
   <property name="toolTip">
    <string>Perform class tests</string>
   </property>
  </action>
  <action name="action_init">
   <property name="text">
    <string>Init</string>
   </property>
  </action>
  <action name="action_numPy">
   <property name="text">
    <string>NumPy</string>
   </property>
  </action>
  <action name="action_algorithmData">
   <property name="text">
    <string>AlgorithmData</string>
   </property>
  </action>
  <action name="action_algorithmInput">
   <property name="text">
    <string>AlgorithmInput</string>
   </property>
  </action>
  <action name="action_charInput">
   <property name="text">
    <string>Char input</string>
   </property>
  </action>
  <action name="action_plot">
   <property name="text">
    <string>plot</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
